Stripe/PublicHeaders/STPPaymentContext.h

```diff
@@ -372,7 +372,20 @@ didCreatePaymentResult:(STPPaymentResult *)paymentResult
             completion:(STPErrorBlock)completion;
 
 /**
- This is invoked by an `STPPaymentContext` when it is finished. This will be called after the payment is done and all necessary UI has been dismissed. You should inspect the returned `status` and behave appropriately. For example: if it's `STPPaymentStatusSuccess`, show the user a receipt. If it's `STPPaymentStatusError`, inform the user of the error. If it's `STPPaymentStatusUserCanceled`, do nothing.
+ Unfortunately, some payment flows support user cancellation after they have initially authorized a payment but before it has been processed.
+ ApplePay is one such payment flow, where the ApplePay sheet remains displayed with a "Cancel" button until the payment is completed.
+ As a result, the following sequence of events may occur:
+    1. The user authorizes the payment
+    2. the payment context object calls your degelate's paymentContext:didCreatePaymentResult
+    3. The user taps the cancel button before the payment is completed
+ In order to handle this case, the payment context object calls this delegate method whenever the user taps ApplePay's cancel button after paymentContext:didCreatePaymentResult. Note that paymentContext:didFinishWithStatus will not be called in this case.
```

Comment:
Same

Reply:
I would also advocate for some line length consistency, even in comments. Having one-off long lines can get kind of confusing.